karate-e2e-tests/src/test/java/accessibility/axe.feature

```diff
@@ -0,0 +1,31 @@
+Feature: axe accessibility native
+
+  Background:
+    * configure driver = { type: 'chrome', showDriverLog: true }
+
+  Scenario:
+    * def axeJs =
```

Comment:
ah. you're gonna love this:

```cucumber
* def axeJs = karate.http('https://cdnjs.cloudflare.com/ajax/libs/axe-core/4.0.2/axe.min.js').get().body
```

try it :)

Reply:
BTW even a GET via karate should have worked, but there is this new JS API which is interesting

Reply:
above was really cool - karate keeps getting sleek!